test/hook/vue-server-renderer.mjs

```diff
@@ -0,0 +1,4 @@
+import { strictEqual } from 'assert'
```

Comment:
What do you think about adding a leading comment that describes the purpose of/reason for this test?

Reply:
Maybe just link to #139?

Reply:
That would at least be easier than needing to navigate `git blame`.